### PR TITLE
[dhctl] feat(dhctl-for-commander): production ready commander mode (part 3)

### PR DESCRIPTION
**Pull request by distorhead**
#### Features

* Check step integrated into the converge.
    * In the commander mode check operation has been integrated into the converge operation.
    * Why do we need it: when running converge it is possible that user has introduced such changes into the DH configuration which will result in destructive changes in cluster. Destructive changes should be approved by the special flow. Check operation on the other hand durably detects destructive changes and calculates so called destructive-change-id.
    * By running check operation just before converge we detect if changes introduced by the user are destructive or not, and if it destructive, then we perform destructive-change-id verification. And only if changes are safe and confirmed by the user, then we perform actual converge operation.

* Added new --connnection-config argument to all dhctl subcommands which have ssh-related params. Using this argument we can pass a special yaml file which contains
    * dhctl.deckhouse.io/v1 SSHConfig and SSHHost resources. It is new alternative way to pass ssh-related params for dhctl.
    * Also added validations related to these new CRDs (more info about validation below).
    * Added ability to specify passphrase for a private key using new connnection-config. To enable this feature ssh-add was rewritten to use builtin golang ssh-agent-client.

* Introduced new 'dhctl server' command. This command starts dhctl in the server mode, which could run dhctl operations by connecting to the dhctl-server over grpc-based protocol.
    * In this mode dhctl could perform multiple operations in parallel. Implementation details: dhctl server on each request would spawn dhctl-subprocess which runs actual operation.
    * This subprocess approach has been implemented due to the lack of ability to execute dhctl operations in multithreaded way in the current implementation. 

* Check operation changes.
    * Check operation now creates terraform plan for abandoned nodes to get more detalization about tf-resources which will be deleted on next converge.
    * Check operation result now contains detalization about destructive changes, and also
  exposes terraform plan. These changes needed to calculate destructive-change-ID and
  show detailization on client-side.
    * Check operation returns DestructiveOutOfSync status when there are abandoned nodes – these will be destroyed in the converge.

* Validations. This PR introduces functions in config package to perform advanced validations of dhctl configuration. 
    * Validate configuration itself:
        * config.ValidateResources
        * config.ValidateInitConfiguration
        * config.ValidateClusterConfiguration
        * config.ValidateProviderSpecificClusterConfiguration
        * config.ValidateStaticClusterConfiguration
    * Validate configuration changes (see more info below)
        * config.ValidateClusterSettingsChanges
    * Ssh private key validation for SSHConfig.
    * Introduced configuration diff validation method: caller should provide current and desired configuration and this method will allow or disallow changes based on rules defined in the resources CRDs.
    * Introduced 3 new annotations for CRDs definition: `x-unsafe-rules: [<ruleName>, ...]`, `x-unsafe: true|false` and `x-rules: [<ruleName>, ...]`.
        * When `x-unsafe` is set then this field, then diff-validation method will simply deny any change of this field (this annotation has priority over `x-usafe-rules`).
        * When `x-usafe-rules` is set to one of the following builtin rules: `updateReplicas`, `deleteZones` or `updateMasterImage`, then diff-validation-method will perform specified custom rule validation:
            * `updateReplicas` denies zero-replicas, denies if replicas decreased from more than 1 replica to single replica (master replicas cannot be changed that way).
            * `deleteZones` denies deleting zones if master replicas are less than 3.
            * `updateMasterImage` denies changing of master imageID if there are less than 3 master replicas.
        * `x-rule` currently supports only `sshPrivateKey` rule, which validates ssh-private key format and passphrase correctness.
    * IMPORTANT NOTICE: validation method config.ValidateClusterSettingsChanges currently used only in commander and it meant to allow or deny changes which converge can actually perform automatically. It is not supposed to use this validation method if some field change can actually be performed but you first need some manual actions before running 'dhctl converge'.

* Introduced new Import operation, which reside in pkg/operations/import and not used in the CLI (same as check pkg/operations/check). Import only allowed to run in commander-mode. Import operation intents to import some deckhouse cluster into commander instance.
    * Import operation consists of the following phases:
        * Scan
            * reads info and configuration from target cluster.
        * Capture
            * changes state of target cluster so that it becomes linked to the commander:
                * create deckhouse-commander-agent and deckhouse-admin ModuleConfigs if not created yet;
                * disable terraform-manager (no auto-converge allowed for commander-mode).
        * Check
            * runs regular check procedure on imported cluster and gives check result to the operation runner.
    * No commander-uuid checks yet.
        * In future version there will be some built-in check to assure cluster not being used by another commander instance.

#### Fixes

* Fix check operation for static cluster failure. Do not run check-infra step in check operation for static cluster, only check configuration in such case.

#### Other / refactor / chore

* config.ValidateDiscoveryData interface changed a little, adjusted all hooks in modules to use new API.

* Create all terraform runners using TerraformContext. Such approach allows sharing terraform runners between different phases of operation if needed.
  TerraformContext becomes the single point where all terraform runners are created and initialized, which also should be convinient by itself for systematization and overview of all runners flags.

* Increased Getting-Resources-List attemts for more durable bootstrap operation (from 2min max to 4min).

* Increased attempts count when creating resources to fix 'connection refused' from validating webhook (from 2min max to 10min).

* Disable locks in commander-mode:
    * local-bootstraper for bootstrap operation.
    * local-converger for converge and bootstrap operations.
    * local-destroyer for destroy operation.

* On-phase-hook in phased-execution-context reworked so that it can accept additional completed phase data argument.

## Why do we need it, and what problem does it solve?

This PR is a third part of changes needed to bootstrap production ready commander version.
This PR introduces new 'dhctl server' subcommand which allows running basic dhctl operations over network.

First part: https://github.com/deckhouse/deckhouse/pull/5893.
Second part: https://github.com/deckhouse/deckhouse/pull/5966

## Why do we need it in the patch release (if we do)?

Not necessarily.

## What is the expected result?

It is not expected that this PR will significantly change dhctl-CLI behaviour.
All additional functions needed for the commander added as options which are disabled by default.
It is expected that after this PR commander module will receive ability to run dhctl-server from official deckhouse installer image.

### CODE REVIEW

File: candi/openapi/dhctl/ssh_host_configuration.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        host: 172.16.0.0
+    properties:
+      apiVersion:
+        type: string
+        description: Version of the Deckhouse API.
+        enum: [dhctl.deckhouse.io/v1]
+      kind:
+        type: string
+        enum: [SSHHost]
+      host:
```
Comment (name212) [inline candi/openapi/dhctl/ssh_host_configuration.yaml:22]: for converge we can pass multiple --ssh-host addresses

File: dhctl/pkg/config/validation.go [hunk 2]
```diff
@@ -21,51 +21,431 @@ import (
...
+// ValidateProviderSpecificClusterConfiguration parses and validates cluster ProviderSpecificClusterConfiguration.
+// It requires one doc with kind in
+// [
+// "OpenStackClusterConfiguration",
+// "AWSClusterConfiguration",
+// "GCPClusterConfiguration",
+// "YandexClusterConfiguration",
+// "VsphereClusterConfiguration",
+// "AzureClusterConfiguration",
+// "VCDClusterConfiguration",
```
Comment (name212) [inline dhctl/pkg/config/validation.go:284]: Zvirt need to add

File: dhctl/pkg/terraform/terraform_context.go [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+					WithAutoApprove(opts.AutoApprove)
+
+				r.WithAdditionalStateSaverDestination(opts.AdditionalStateSaverDestinations...)
+
+				tomb.RegisterOnShutdown("base-infrastructure", r.Stop)
+				return r
+			} else {
+				r := NewImmutableRunnerFromConfig(metaConfig, "base-infrastructure").
+					WithVariables(metaConfig.MarshalConfig()).
+					WithAutoApprove(true)
```
Comment (name212) [inline dhctl/pkg/terraform/terraform_context.go:69]: Is it required for autoapprove == true, Looks like it allow potential destructive operation

File: dhctl/pkg/config/validation_test.go [hunk 4]
```diff
@@ -17,311 +17,531 @@ package config
...
+		"ok": {
+			config: `
+---
+---
+# https://deckhouse.ru/documentation/v1/installing/configuration.html#initconfiguration
+apiVersion: deckhouse.io/v1
+kind: InitConfiguration
+deckhouse:
+  imagesRepo: registry.deckhouse.io/deckhouse/ee
+  registryDockerCfg: eyJhdXRocyI6eyJyZWdpc3RyeS5kZWNraG91c2UuaW8iOnsiYXV0aCI6ImJHbGpaVzV6WlMxMGIydGxianBtZWxkeFMzZGxOR2s0VEU0ME5tUmtNbGxWTWxKWmJYTkNXVmcyV25sTVJ3PT0ifX19
```
Comment (name212) [inline dhctl/pkg/config/validation_test.go:103]: valid license key in tests

File: dhctl/pkg/config/validation_test.go [hunk 5]
```diff
@@ -17,311 +17,531 @@ package config
...
+  enabled: true`,
+			errContains: `ValidationFailed: exactly one "InitConfiguration" required`,
+		},
+		"multiple init configs": {
+			config: `
+apiVersion: deckhouse.io/v1
+kind: InitConfiguration
+deckhouse:
+  imagesRepo: registry.deckhouse.io/deckhouse/ee
+  registryDockerCfg: 1eyJhdXRocyI6eyJyZWdpc3RyeS5kZWNraG91c2UuaW8iOnsiYXV0aCI6ImJHbGpaVzV6WlMxMGIydGxianBtZWxkeFMzZGxOR2s0VEU0ME5tUmtNbGxWTWxKWmJYTkNXVmcyV25sTVJ3PT0ifX19
```
Comment (name212) [inline dhctl/pkg/config/validation_test.go:143]: and here

Comment (name212) [reply dhctl/pkg/config/validation_test.go:143]: https://github.com/deckhouse/deckhouse/pull/7157/files#r1577985417

File: dhctl/pkg/config/validation_test.go [hunk 6]
```diff
@@ -17,311 +17,531 @@ package config
...
+  releaseChannel: Alpha
+---
+apiVersion: deckhouse.io/v1
+kind: InitConfiguration
+deckhouse:
+  imagesRepo: registry.deckhouse.io/deckhouse/ee
+  registryDockerCfg: 2eyJhdXRocyI6eyJyZWdpc3RyeS5kZWNraG91c2UuaW8iOnsiYXV0aCI6ImJHbGpaVzV6WlMxMGIydGxianBtZWxkeFMzZGxOR2s0VEU0ME5tUmtNbGxWTWxKWmJYTkNXVmcyV25sTVJ3PT0ifX19
```
Comment (name212) [inline dhctl/pkg/config/validation_test.go:150]: here
https://github.com/deckhouse/deckhouse/pull/7157/files#r1577985417

File: dhctl/pkg/config/validation_test.go [hunk 7]
```diff
@@ -17,311 +17,531 @@ package config
...
+  releaseChannel: Stable`,
+			errContains: `ValidationFailed: exactly one "InitConfiguration" required`,
+		},
+		"extra kinds": {
+			config: `
+apiVersion: deckhouse.io/v1
+kind: InitConfiguration
+deckhouse:
+  imagesRepo: registry.deckhouse.io/deckhouse/ee
+  registryDockerCfg: eyJhdXRocyI6eyJyZWdpc3RyeS5kZWNraG91c2UuaW8iOnsiYXV0aCI6ImJHbGpaVzV6WlMxMGIydGxianBtZWxkeFMzZGxOR2s0VEU0ME5tUmtNbGxWTWxKWmJYTkNXVmcyV25sTVJ3PT0ifX19
```
Comment (name212) [inline dhctl/pkg/config/validation_test.go:160]: https://github.com/deckhouse/deckhouse/pull/7157/files#r1577985417

File: dhctl/pkg/kubernetes/actions/converge/check.go [hunk 8]
```diff
@@ -52,46 +58,138 @@ type NodeGroupCheckResult struct {
...
+	}
 
-	return terraform.CheckPipeline(nodeRunner, nodeName)
+	var stateSavers []terraform.SaverDestination
+	if opts.CommanderMode {
+		stateSavers = append(stateSavers, NewNodeStateSaver(kubeCl, nodeName, nodeGroupName, nil))
+	}
+	nodeRunner := terraformContext.GetCheckNodeDeleteRunner(cfg, terraform.NodeDeleteRunnerOptions{
+		AutoDismissDestructive:           false,
+		AutoApprove:                      true,
```
Comment (name212) [inline dhctl/pkg/kubernetes/actions/converge/check.go:131]: https://github.com/deckhouse/deckhouse/pull/7157/files#r1578002361

File: candi/openapi/dhctl/ssh_host_configuration.yaml (hunk 1 above)
Comment (distorhead) [reply candi/openapi/dhctl/ssh_host_configuration.yaml:22]: @name212 In such case we simply pass multiple SSHHost resources.

File: dhctl/pkg/config/validation.go (hunk 2 above)
Comment (distorhead) [reply dhctl/pkg/config/validation.go:285]: Fixed.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/eks'...

**Comment by z9r5** [comment]
@distorhead Hi! Don't forget to delete the failed clusters, please.  JFI

**Comment by distorhead** [comment]
/e2e/abort static;Static;containerd;1.25 8960580890 8960580890-1-con-1-25 /sys/deckhouse-oss/install:pr7157 astra@192.168.199.93

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by distorhead** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.25 8960581011 8960581011-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by distorhead** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.25 8960581012 8960581012-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by distorhead** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.25 8960581067 8960581067-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by distorhead** [comment]
/e2e/abort azure;Standard;containerd;1.25 8960581120 8960581120-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by distorhead** [comment]
/e2e/abort openstack;Standard;containerd;1.25 8960581090 8960581090-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by distorhead** [comment]
/e2e/abort vsphere;Standard;containerd;1.25 8960581093 8960581093-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by distorhead** [comment]
/e2e/abort eks;WithoutNAT;containerd;1.25 8960581163 8960581163-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by distorhead** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.25 8967578123 8967578123-1-con-1-25 /sys/deckhouse-oss/install:pr7157 redos@158.160.108.13

**Comment by distorhead** [comment]
/e2e/abort azure;Standard;containerd;1.25 8967578213 8967578213-1-con-1-25 /sys/deckhouse-oss/install:pr7157 azureuser@4.231.89.112

**Comment by distorhead** [comment]
/e2e/abort openstack;Standard;containerd;1.25 8967578498 8967578498-1-con-1-25 /sys/deckhouse-oss/install:pr7157 redos@188.124.39.227

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by distorhead** [comment]
/e2e/abort vsphere;Standard;containerd;1.25 8967578745 8967578745-1-con-1-25 /sys/deckhouse-oss/install:pr7157 ubuntu@10.220.203.26

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by distorhead** [comment]
/e2e/abort vsphere;Standard;containerd;1.25 8971596779 8971596779-1-con-1-25 /sys/deckhouse-oss/install:pr7157 ubuntu@10.220.203.26

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by distorhead** [comment]
/e2e/abort vsphere;Standard;containerd;1.25 8971690714 8971690714-1-con-1-25 /sys/deckhouse-oss/install:pr7157 ubuntu@10.220.203.36

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by distorhead** [comment]
/e2e/abort vsphere;Standard;containerd;1.25 8972258368 8972258368-1-con-1-25 /sys/deckhouse-oss/install:pr7157 ubuntu@10.220.203.26

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by distorhead** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.25 8975392647 8975392647-1-con-1-25 /sys/deckhouse-oss/install:pr7157 redos@84.201.174.48

**Comment by distorhead** [comment]
/e2e/abort eks;WithoutNAT;containerd;1.25 8975392403 8975392403-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by distorhead** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.25 8975392376 8975392376-1-con-1-25 /sys/deckhouse-oss/install:pr7157 centos@35.158.10.148

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by distorhead** [comment]
/e2e/abort static;Static;containerd;1.25 8975392318 8975392318-1-con-1-25 /sys/deckhouse-oss/install:pr7157 astra@192.168.199.177

**Comment by distorhead** [comment]
/e2e/abort openstack;Standard;containerd;1.25 8975391910 8975391910-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/openstack'...

**Comment by distorhead** [comment]
/e2e/abort eks;WithoutNAT;containerd;1.25 8984734863 8984734863-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by distorhead** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.25 8986228924 8986228924-1-con-1-25 /sys/deckhouse-oss/install:pr7157 centos@18.197.3.163

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/eks'...

**Comment by distorhead** [comment]
/e2e/abort eks;WithoutNAT;containerd;1.25 8987567817 8987567817-1-con-1-25 /sys/deckhouse-oss/install:pr7157

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @distorhead. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by z9r5** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.25 8967577995 8967577995-1-con-1-25 /sys/deckhouse-oss/install:pr7157 user@34.107.38.36

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by z9r5** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.25 8967577998 8967577998-1-con-1-25 /sys/deckhouse-oss/install:pr7157 centos@3.125.111.142

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Review by name212 — APPROVED** [review]

**Pull request merged**